src/logger.h

```diff
@@ -1,4 +1,28 @@
 #pragma once
+
+#include <fstream>
+
 #include <boost/log/trivial.hpp>
+#include <boost/log/utility/setup/from_stream.hpp>
+#include <boost/log/core/core.hpp>
+#include <boost/log/sources/logger.hpp>
+
+namespace logging = boost::log;
+namespace src = boost::log::sources;
+using LoggerType = src::logger_mt;
+
+#define CREATE_LOCAL_LOGGER(logger) \
+  src::logger_mt logger
+
+#define CREATE_GLOBAL_LOGGER(logger) \
+namespace { \
+  GET_LOCAL_LOGGER(); \
```

Comment:
@malirod you have several time mentioned that boos logger has no name. Not sure if I understand you.

1. Boost suggests to avoid global logger at all. Although there is an ability to create one.
2. Boost suggest to use logger as aclass member.
3. Boost logger has a names. It registeres and uses these names from within `log::core`